packages/contracts/hardhat.config.ts

```diff
@@ -105,7 +105,7 @@ const config: HardhatUserConfig = {
       throwOnTransactionFailures: true,
       throwOnCallFailures: true,
       blockGasLimit: BigNumber.from(10).pow(6).mul(300).toNumber(), // 300 million, really high to test some things that are only possible with a higher block gas limit
-      gasPrice: BigNumber.from(10).pow(9).mul(150).toNumber(), // 150 gwei
+      gasPrice: BigNumber.from(10).pow(9).mul(300).toNumber(), // 300 gwei
```

Comment:
if this number changes because we are doing forks,
i recommend using a default like 300, but it could be overridden via .env

Reply:
Yes, makes sense. Let's change this in the `osx-plugin-template` and sync these changes with forks. Do you want to create a task for it?

Reply:
On second thought:

```ts
hardhat: {
  blockGasLimit: BigNumber.from(10).pow(6).mul(300).toNumber(), // 300 million, really high to test some things that are only possible with a higher 
block gas limit
  gasPrice: BigNumber.from(10).pow(9).mul(300).toNumber(), // 300 gwei
  ),
  ...
},
```
since these values are only for the hardhat network, we might be fine and can just specify a very high `gasPrice` value.

Reply:
if the error is clear when gas price is low, then setting a high value is fine,
I just don't want engineers spend time on debugging